src/dev/src/app/radios/radios.demo.html

```diff
@@ -33,32 +38,42 @@ <h4>Vertical</h4>
       <div class="clr-radio-wrapper">
         <input type="radio" name="radio2" id="vertical-radio4" placeholder="Enter value here" class="clr-radio">
         <label for="vertical-radio4" class="clr-control-label">option 1</label>
+      </div>
+      <div class="clr-radio-wrapper">
         <input type="radio" name="radio2" id="vertical-radio5" placeholder="Enter value here" class="clr-radio">
         <label for="vertical-radio5" class="clr-control-label">option 2</label>
+      </div>
+      <div class="clr-radio-wrapper">
         <input type="radio" name="radio2" id="vertical-radio6" placeholder="Enter value here" class="clr-radio">
         <label for="vertical-radio6" class="clr-control-label">option 3</label>
       </div>
-      <!-- IMPORTANT DIFFERENCE IN STRUCTURE! ICON IS NOT PART OF THE INPUT WRAPPER -->
-      <clr-icon class="clr-validate-icon" shape="exclamation-circle"></clr-icon>
-      <span class="clr-subtext">Helper Text</span>
+      <div class="clr-subtext-wrapper">
+        <clr-icon class="clr-validate-icon" shape="exclamation-circle"></clr-icon>
+        <span class="clr-subtext">Helper Text</span>
+      </div>
     </div>
   </div>
 
   <div class="clr-form-control">
     <label class="clr-control-label">
       Inline radio</label>
-    <div class="clr-control-container">
-      <div class="clr-radio-wrapper clr-radio-inline">
+    <div class="clr-control-container clr-control-inline">
+      <div class="clr-radio-wrapper">
```

Comment:
So we now require a `.clr-radio-wrapper` around each radio and input?

Is this a breaking change?

Reply:
This is actually what is in the docs, the demo just didn't have it right.